vendor/DOtherSide/lib/src/DOtherSide.cpp

```diff
@@ -1476,42 +1475,6 @@ char *dos_escape_html(char* input)
    return convert_to_cstring(QString(input).toHtmlEscaped().toUtf8());
 }
 
-char *dos_image_resizer(const char* imagePathOrData, int maxSize, const char* tmpDirPath)
-{
-    const auto base64JPGPrefix = "data:image/jpeg;base64,";
-    QImage img;
-    bool loadResult = false;
-
-    // load the contents
-    if (qstrncmp(base64JPGPrefix, imagePathOrData, qstrlen(base64JPGPrefix)) == 0)  { // binary BLOB
-      loadResult = img.loadFromData(QByteArray::fromBase64(QByteArray(imagePathOrData).mid(qstrlen(base64JPGPrefix))));  // strip the prefix, decode from b64
-    } else { // local file or URL
-      const auto localFileOrUrl = QUrl::fromUserInput(imagePathOrData); // accept both "file:/foo/bar" and "/foo/bar"
-      if (localFileOrUrl.isLocalFile()) {
-        loadResult = img.load(localFileOrUrl.toLocalFile());
-      } else {
-        QEventLoop loop;
```

Comment:
Using a nested event loop is highly discouraged, even more so this is called from with an async NIM job, so we decided to drop this path @jrainville 

Frankly the only native FileDialog that supports selecting remote URLs is the one in Linux/KDE so I'm fine with this :)

Reply:
All the other platforms/OSs will let the user select local files only anyway

Reply:
Did I write this code? If I did according to the blame, it might just be because I moved it? I have no recollection of this code :sweat_smile: